book/src/drivers.md

```diff
@@ -74,7 +74,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [MacroSAN](https://github.com/macrosan-csi/macrosan-csi-driver) | `csi-macrosan` | v1.0 | A Container Storage Interface (CSI) Driver for MacroSAN Block Storage | Persistent | Read/Write Single Pod | Yes | 
 [Manila](https://github.com/kubernetes/cloud-provider-openstack/tree/master/pkg/csi/manila) | `manila.csi.openstack.org` | v1.1, v1.2 | A Container Storage Interface (CSI) Driver for OpenStack Shared File System Service (Manila) | Persistent  | Read/Write Multiple Pods | Yes | Snapshot, Topology
 [MooseFS](https://github.com/moosefs/moosefs-csi) | `com.tuxera.csi.moosefs` | v1.0 | A Container Storage Interface (CSI) Driver for [MooseFS](https://moosefs.com/) clusters. | Persistent | Read/Write Multiple Pods | Yes | 
-[NetApp](https://github.com/NetApp/trident) | `csi.trident.netapp.io` | v1.0, v1.1, v1.2, v1.3 | A Container Storage Interface (CSI) Driver for NetApp's [Trident](https://netapp-trident.readthedocs.io/) container storage orchestrator | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Cloning, Topology
+[NetApp](https://github.com/NetApp/trident) | `csi.trident.netapp.io` | v1.0, v1.1, v1.2, v1.3, v1.4, v1.5 | A Container Storage Interface (CSI) Driver for NetApp's [Trident](https://docs.netapp.com/us-en/trident/index.html) container storage orchestrator | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Cloning, Topology
```

Comment:
I wonder if the version would better be expressed as a range instead of a list (v1.0-v1.5). Eventually the list will be too long and need abbreviation, but I don't know if we're there yet.

Reply:
I followed what the other drivers are doing currently. I think this is a good suggestion though.